rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly add the book and person to the getter and setter (attr_accessor). This is how it is supposed to be ```attr_accessor :date, :book, :person```